DESCRIPTION

```diff
@@ -1,15 +1,20 @@
-Package: KarsBackend
-Title: Provides Backend Functions To KARS
-Version: 0.0.0.9000
+Package: histoc
+Title: Histocompatibility functios for organ transplantation
+Version: 0.0.0.9999
 Authors@R: 
-    person("Bruno", "Lima", , "balima78@gmail.com", role = c("aut", "cre"),
+    person("Bruno", "Lima", , "balima78@gmail.com", role = "aut",
            comment = c(ORCID = "YOUR-ORCID-ID"))
-Description: What the package does (one paragraph).
-License: `use_mit_license()`, `use_gpl3_license()` or friends to pick a
-    license
+Description: This package contains functions to use on organ transplantation. Histocompatibility analysis are possible with information from donors and candidates.
+License: `use_mit_license()`
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.1.2
-Suggests: 
+Suggests:
     testthat (>= 3.0.0)
+Imports: 
```

Comment:
We need to create a ticket to look into how other people enforce packages to be installed and loaded.

Reply:
I'll solve this myself.